src/Evolution/Systems/ScalarWave/System.hpp

```diff
@@ -20,7 +20,18 @@ class Variables;
 
 /*!
  * \ingroup EvolutionSystemsGroup
- * \brief Items related to evolving the scalar wave equation:
+ * \brief Items related to evolving the scalar wave equation.
+ *
+ * The equations of motion for the system augmented with constraint damping
+ * terms are given by Eq. (15), (23) and (24) of \cite Holst2004wt (setting
+ * background spacetime to Minkowskian):
+ * \f{align*}
+ * \partial_t \psi =& -\Pi \\
+ * \partial_t \Pi  =& -\partial^i \Phi_i \\
+ * \partial_t \Phi_i =& -\partial_i \Pi + \gamma_2 (\partial_i \psi - \Phi_i)
+ * \f}
+ * In our implementation here, though, we disable the constraint damping terms
```

Comment:
I'm a bit confused by the last sentence. I thought you're adding the ability to have non-zero constraint damping. If so, then I think what you mean is `To disable constraint damping, set gamma_2 to zero`